modules/RTC/JitsiLocalTrack.js

```diff
@@ -320,6 +335,99 @@ export default class JitsiLocalTrack extends JitsiTrack {
         }
     }
 
+    /**
+     * Starts or stops the effect process and returns the corresponding stream.
+     *
+     * @private
+     * @param {Boolean} enable - Used to start or stop the effect processing
+     * @param {*} effect - Optional parameter represents effect class to create instance
+     * @returns {MediaStream}
+     */
+    _streamEffectEnable(enable: Boolean, effect) {
```

Comment:
Since the code is totally different for `enable===true` and `enable===false` I think it makes sense to have to methods. Maybe `_startStreamEffect` and `_stopStreamEffect`. WDYT?

Reply:
Create the two methods _startStreamEffect and _stopStreamEffect